docs/README.md

```diff
@@ -68,18 +68,18 @@ In order to apply CSS customizations in this and other blocks, follow the instru
 
 | CSS Handles               |
 | ------------------------- |
-| `container`               |
-| `formContainer`           |
-| `loginLink`               |
-| `reviewCommentsContainer` |
-| `reviewsHeading`          |
-| `star--empty`             |
-| `star--filled`            |
-| `star`                    |
-| `starpicker`              |
-| `stars`                   |
-| `summaryContainer`        |
-| `writeReviewContainer`    |
+| 'container'               |
+| 'writeReviewContainer'    |
+| 'loginLink'               |
+| 'reviewsRating'           |
+| 'starsContainer'          |
+| 'reviewsHeading'          |
+| 'reviewsRatingAverage'    |
+| 'reviewsRatingCount'      |
+| 'reviewCommentsContainer' |
+| 'reviewComment'           |
+| 'reviewCommentRating'     |
+| 'reviewCommentUser'       |
```

Comment:
It's better to keep the list sorted and with code formatting. Also some existing CSS Handles are missing.

```suggestion
| `container`               |
| `loginLink`               |
| `reviewComment`           |
| `reviewCommentRating`     |
| `reviewCommentsContainer` |
| `reviewCommentUser`       |
| `reviewsHeading`          |
| `reviewsRating`           |
| `reviewsRatingAverage`    |
| `reviewsRatingCount`      |
| `star--empty`             |
| `star--filled`            |
| `star`                    |
| `starpicker`              |
| `starsContainer`          |
| `writeReviewContainer`    |
```